docs/api-swagger-shorter.yaml

```
@@ -58,7 +58,7 @@ paths:
         - Kubernetes Resources
       summary: Create a new resource
       description: Creates a new resource of the specified type.
-      operationId: createResource
+      operationId: createResources
```

```suggestion
      operationId: createResource
```
Na moje to lepiej jest w pojedyńczej formie, bo to tworzy jeden zasób, plus nawet jeżeli to zmienić tą nazwę trzeba by było jeszcze w skryptach, żeby była zgodna